src/libs/Navigation/Navigation.js

```diff
@@ -98,6 +98,11 @@ function dismissModal(shouldOpenDrawer = false) {
     openDrawer();
 }
 
+function isActive(routePath) {
```

Comment:
This needs a JS doc for the param and return value. Also could use a brief explanation about how we should use this.